packages/toolkit/src/codewhisperer/util/dependencyGraph/dependencyGraph.ts

```diff
@@ -153,18 +152,6 @@ export abstract class DependencyGraph {
         return dir + extension
     }
 
-    protected removeDir(dir: string) {
-        if (existsSync(dir)) {
-            fs.removeSync(dir)
-        }
-    }
-
-    protected removeZip(zipFilePath: string) {
-        if (existsSync(zipFilePath)) {
-            fs.unlinkSync(zipFilePath)
```

Comment:
Just for future reference: checking for existence is a race condition anyway. So it's better to just handle the error instead of checking first.

Reply:
Good point, ended up having to update this PR to do this